smrf/framework/changelog.ini

```diff
@@ -22,8 +22,8 @@ stations -> REMOVED
 # Gridded
 gridded/file -> gridded/wrf_file
 gridded/directory -> gridded/hrrr_directory
-gridded/zone_number -> REMOVED
-gridded/zone_letter -> REMOVED
+; gridded/zone_number -> REMOVED
+; gridded/zone_letter -> REMOVED
```

Comment:
Are these left commented for posterity?

Reply:
This will be address with issue #100 so I was just leaving it in as it will go back into the change log